rtrlib/rtr_mgr.h

```diff
@@ -288,7 +290,7 @@ const char *rtr_mgr_status_to_str(enum rtr_mgr_status status);
  * @param[in] fp Pointer to callback function with signature \c pfx_for_each_fp.
  * @param[in] data This parameter is forwarded to the callback function.
  */
-void rtr_mgr_for_each_ipv4_record(struct rtr_mgr_config *config, pfx_for_each_fp fp, void *data);
+void rtr_mgr_for_each_ipv4_record(struct rtr_mgr_config *config, rtr_pfx_for_each_fp fp, void *data);
```

Comment:
maybe prefix this kind of functions, which subsystem they will access.

so something like: `rtr_mgr_roa_for_each_ipv4_record`